### PR TITLE
added chiado updates

### LINKED ISSUE #819: Prepare the main docs for the Chiado update
**TO DO 1** -- these changes are temporarily saved in a local file

- Add a guide for joining the new network -- done
- Go through all Buenavista mentions and replace if needed -- done
- Mention the transition period -- done
- Mention the new denomination in Glossary and Create a validator -- done
- Update the Chiado guide with correct details
    - The new faucet address -- done
    - The correct binary release -- done
    - The links to the release, endpoints, persistent peers -- done

**TO DO 2** -- closer to the release

 - Move these updates to a new branch -- done
 - Replace `buenavista-1` / `chiado-1337-1` with ` chiado_10010-1` -- done
- Check again for Buenavista mentions -- done
- Add announcements to Join Buenavista, Join Chiado, and Create a validator -- done
- Add notes about the denomination -- done
- Add an explanation to the GMP guide why it uses Buenavista -- done

**TO DO 3** -- check facts

- Is this the branch `--branch v0.5.0` correct? -- change to v0.5.1
- Is this correct? `https://rpc.chiado.wardenprotocol.org:443` -- yes
- Is the command for getting the block height correct? -- yes
- Amounts in the Create a validator guide -- yes, we should ad 12 zeroes

**TO DO 4** -- right before the publication

- Resolve conflicts with main** -- done
- Revisit Join Chiado and make it consistent with the current version of Buenavista -- done
- Make sure that the list of persistent peers and the list of endpoints are available -- done
- **Check the [list of seeds](https://github.com/warden-protocol/networks/blob/main/testnets/chiado/chain.json#L68-L69) -- unless we implement a dynamic list by then**

### PR DESCRIPTION
**Pull request by ijonele**
## Summary by CodeRabbit

- **New Features**
  - Introduced a comprehensive tutorial for joining the Chiado testnet, detailing installation and configuration steps.

- **Changes**
  - Updated documentation to reflect the transition from the Buenavista testnet to the Chiado testnet across multiple sections.
  - Changed URLs and references in various documents to point to the Chiado testnet.
  - Enhanced clarity and detail in instructions for operating nodes and joining testnets.
  - Added a "Next steps" section in the local chain operation guide for further actions.

- **Chores**
  - Updated spelling file with new terms relevant to the documentation, including "Evmos" and "Chiado."

### CODE REVIEW

**Review by alijnmerchant21 — DISMISSED** [review]
Looks good to me so far. My only concern is links like rpc, snapshot, faucet will stay same.

File: docs/developer-docs/docs/operate-a-node/chiado-testnet/join-chiado.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+- [Install Go](https://golang.org/doc/install) 1.22.3 or later.
+- If you wish to build the binary from the source code, [install just](https://github.com/casey/just) 1.34.0 or later.
+
+## 1. Install
+
+To be able to interact with the node, install `wardend` (the Warden binary) using the script below. There are two ways to do it:
+
+### Option 1: Use the prebuilt binary
+
+1. Download the binary for your platform from the [release page](https://github.com/warden-protocol/wardenprotocol/releases/tag/v0.5.1) and unzip it. The archive contains the `wardend` binary.
```
Comment (jlehtimaki) [inline docs/developer-docs/docs/operate-a-node/chiado-testnet/join-chiado.md]:
```suggestion
1. Download the binary for your platform from the [release page](https://github.com/warden-protocol/wardenprotocol/releases/tag/v0.5.2) and unzip it. The archive contains the `wardend` binary.
```

File: docs/developer-docs/docs/operate-a-node/chiado-testnet/join-chiado.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+- The chain ID in queries: `chiado_10010-1`
+- Denomination: `award` / 0.000000000000000001 WARD
+- Endpoints: [networks repository > chiado](https://github.com/warden-protocol/networks/tree/main/testnets/chiado)
+- The current `wardend` version: **v0.5.1**
+
+## Version history
+
+| Release |
+| ------- |
+| v0.5.1  |
```
Comment (jlehtimaki) [inline docs/developer-docs/docs/operate-a-node/chiado-testnet/join-chiado.md]:
```suggestion
| v0.5.2  |
```

File: docs/developer-docs/docs/operate-a-node/chiado-testnet/join-chiado.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+
+## Prerequisites
+
+Before you start, complete the following prerequisites:
+
+- We recommend running public testnet nodes on machines with the following characteristics:
+  - at least 8 cores
+  - 32GB of RAM
+  - 300GB of disk space
```
Comment (jlehtimaki) [inline docs/developer-docs/docs/operate-a-node/chiado-testnet/join-chiado.md]:
```suggestion
  - at least 4 cores
  - 16GB of RAM
  - 300GB of disk space
```

File: docs/developer-docs/docs/operate-a-node/chiado-testnet/join-chiado.md [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+:::
+
+## Overview
+
+This tutorial explains how to run the Warden binary, `wardend`, and join the **Chiado testnet**:
+
+- The chain ID in queries: `chiado_10010-1`
+- Denomination: `award` / 0.000000000000000001 WARD
+- Endpoints: [networks repository > chiado](https://github.com/warden-protocol/networks/tree/main/testnets/chiado)
+- The current `wardend` version: **v0.5.1**
```
Comment (jlehtimaki) [inline docs/developer-docs/docs/operate-a-node/chiado-testnet/join-chiado.md]:
```suggestion
- The current `wardend` version: **v0.5.2**
```

File: docs/developer-docs/docs/operate-a-node/chiado-testnet/join-chiado.md [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+   :::tip
+   When interacting with the node, you should add the path to the binary before `wardend` [commands](/operate-a-node/node-commands).
+   :::
+
+### Option 2: Use the source code
+
+1. Clone the repository and navigate to the root directory:
+
+   ```bash
+   git clone --depth 1 --branch v0.5.1 https://github.com/warden-protocol/wardenprotocol
```
Comment (jlehtimaki) [inline docs/developer-docs/docs/operate-a-node/chiado-testnet/join-chiado.md]:
```suggestion
   git clone --depth 1 --branch v0.5.2 https://github.com/warden-protocol/wardenprotocol
```

**Review by jlehtimaki — CHANGES_REQUESTED** [review]

**Review by jlehtimaki — APPROVED** [review]

**Pull request merged**